apps/public/src/app/(map)/locatie/kies/page.module.css

```
@@ -0,0 +1,58 @@
+.page {
+  grid-template-rows: auto 1fr auto;
+  height: 100vh;
+}
+
+.assetList {
+  display: none;
+  grid-row-start: 2;
+  position: relative;
+
+  @media screen and (min-width: 68rem) {
+    display: block;
+  }
+}
+
+.toggleAssetList {
```

Maybe something like this, to align it with the prop name? 

```suggestion
.showAssetList {
```

---

sure